post_to_github.sh

```diff
@@ -0,0 +1,9 @@
+rid=$(curl -H "Content-Type: application/json" -H "Authorization: token $1" -X POST -d '{"tag_name":"v'$(cat VERSION)'", "target_commitish":"env/dev-main","name":"v'$(cat VERSION)'","body":"", "draft":false, "prerelease":false}' https://api.github.com/repos/goshippo/shippo-java-client/releases | jq '.id')
+
+curl https://oss.sonatype.org/content/repositories/releases/com/goshippo/shippo-java-client/v$(cat VERSION)/shippo-java-client-v$(cat VERSION)-javadoc.jar > shippo-java-client-v$(cat VERSION)-javadoc.jar
```

Comment:
I'm not completely sure that the content will be available in the repository immediately, it could take a bit of time for it to show up, we can check

Reply:
We can also use the compiled files in the target folder. Im not sure of the paths to the jar files locally, though.